### ISSUE TITLE
Ideas on improving AGM_Medical

### ISSUE THREAD

**Issue by lubomirblazekcz** [KoffeinFlummi/AGM#1829]
Well now, when was AGM_Medical rewritten, what about expand it and improve it little more? I have some interesting ideas. I don't want something so complicated as has CSE, but I would welcome if AGM would be little more challenging.

**1. Reorganized menu**
So that Left arm is on left, right arm on right etc. it's more intuitive.
![agm-medic](https://cloud.githubusercontent.com/assets/6872956/5537312/ae389c8a-8a9f-11e4-9ca2-1c03bb34ce1a.jpg)

**2. Pressure bandage for heavy injuries**
To bandage heavy injury with one bandage, pressure bandage would be needed. When player has both types of bandages, the additional submenu would be opened upon bandaging.
eg. Bandage left arm, select bandage, bandaging

**3. Bleedout ?**
Not sure if it's possible now to die on low blood. I got the feeling that you will only go unconscious now. But it would be nice addition to bleedout if your injuries are not treated in time. And what about add timer in unconscious, how much blood left? Maybe as option in module.

**4. No reaction on epinephrine**
Now everyone is applying epinephrine, even that the person has heavy injuries, heavy pain, lost of blood etc. Would be good to implement chance to no reaction on epinephrine, so that medic have to actually treat the patient first, to have insurance that the patient will react on epinephrine.

**5. Tourniquet**
Maybe it's not needed, I don't know. But what about to slow down the bleeding on limbs with heavy injuries? Or use it as insurance that bandaging will not fail with heavy injuries. (lots of blood)

I think that **2**, **3** and **4** could for sure improve the experience for medics and for players. 
Maybe this issue can be used as discussion for what could be implemented? Or everyone could add some more ideas?

**Comment by Pakislav2** [comment]
@Sparfell. Indeed, when unconscious there's no way of telling weather someone is in pain, and not necessary unless performing surgery/reviving with epipen.

@Evro, that menu is a great idea. I was confused as to why someone would make a mini-game out of locating the proper body part in the menu. Alas, the right leg/arm should be on the right side of the menu/screen for intuitive location. Body position is irrelevant as you can stand at their head, feet or side.

@Devs.
Why not take it all the way? The medical system, as is, seems half-finished to me. How does a bandage heal wounds? The consensus with which I agree is that the medic should have a special job, but common soldiers should be able to use epis and morphine.

The entire current functionality should remain as is, available to everybody. But bandaging should only stop bleeding. 

1.)To cure the wounds, a medic would have to perform surgery using a surgical kit only they can use. It should take some time, which will make bandaging first important if bleedout is a concern. 

2.)Using morphine second, to stop the pain and thus the patient interrupting the surgical procedure by wiggling(overdose animation) due to pain if conscious would also be important. 

3.)Using epipen should take place after applying morphine and bandaging, so that the target doesn't bleed out (epi increases blood flow/bleeding out) and doesn't pass out again due to pain.

Basically what ACE did, and did well. 
Optional.)Tourniquets aren't important, but it makes sense to use them during a surgical procedure, since it's hard to operate over a bandage. Performing surgery would then re-start bleeding if patient was bandaged before and tourniquet would be necessary to stabilize him until he can be bandaged again after the procedure.

**Comment by esteldunedain** [comment]
@evromalarkey, @Pakislav2 

Some of the things you proposed here are being implemented on #1899

> 1. Reorganized menu
> So that Left arm is on left, right arm on right etc. it's more intuitive.

Done

> 3. Bleedout ?
> Not sure if it's possible now to die on low blood. I got the feeling that you will only go unconscious now. But it would be nice addition to bleedout if your injuries are not treated in time. And what about add timer in unconscious, how much blood left? Maybe as option in module.

Dying from bleeout was already possible. It took between 5 to 8 minutes to loose enough blood to die from that. Furthermore, it does not happen if prevent death while unconscious is set, which I still want to change,

> 3.)Using epipen should take place after applying morphine and bandaging, so that the target doesn't bleed out (epi increases blood flow/bleeding out) and doesn't pass out again due to pain.

Now bleeding rate is strongly increased for about a minute after inyecting epinephrine, so there's much more incentive to bandage pacients first.

It also increases due to excersion (running around, crawling, etc), so pacients shouldn't try to move to much when wounded. Finally, as the pacient loses it's blood it has increasing chance of passing out due to excersion, so they should avoid running first and even moving at all when they are very very low on blood.

**Comment by evromalarkey** [comment]
:+1: looks good
